code/game/machinery/biogenerator.dm

```diff
@@ -55,6 +57,26 @@
 	object = /obj/item/reagent_containers/food/snacks/soywafers
 	cost = 150
 
+/singleton/biorecipe/food/mre
+	name = "Meals Ready to Eat"
+	object = /obj/random/mre
+	cost = 450
+
```

Comment:
I think the biogenerator should only generate food ingredients or whatever, stuff that can be used to make food.
Not like, actual food that is ready to eat from the start, like MREs.